CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+# Project Name and start working in project scope (some things just work until now, some just work after this command)
+project(SDL2_image C)
+
+# Add conan dependencies
+include(../conanbuildinfo.cmake) # TODO This is actually ugly isnt it? Since it only works inside a "require", not using a normal "mkdir build && cd build && conan install .. && cmake .."
```

Comment:
I tried before, but it wasn't working then for me anymore, not finding the include.. weird